src/Loop.php

```diff
@@ -295,11 +295,14 @@ public static function disable($watcherId)
      * Cancel a watcher.
      *
      * This will detatch the event loop from all resources that are associated to the watcher. After this operation the
-     * watcher is permanently invalid. Calling this function MUST NOT fail, even if passed an invalid watcher.
+     * watcher is permanently invalid. Calling this function MUST NOT fail if the passed watcher was issued by this
+     * driver but is no longer valid.
```

Comment:
This block refers to "this driver", but this doesn't really make sense in the context of `Loop`. Not sure about this since currently `Loop` and `Driver` have the same doc blocks.